### DISCUSSION TITLE
Loading Plugin Show missing parameters

### DISCUSSION THREAD

**Discussion by msosa360** [quasarframework/quasar#9596] [General - Components / Directives / etc] [answered]
Hello. I'm using Quasar v v2.0.0-beta.19, and according to the documentation, the Loading plugin can be used this way:

$q.loading.show({
    message: 'Doing something. Please wait...',
    boxClass: 'bg-grey-2 text-grey-9',
    spinnerColor: 'primary'
  })

The problem is that "boxClass" parameter is not defined in the show method (this is que library code):
show (opts? : {
            /**
             * Wait a number of millisecond before showing; Not worth showing for 100ms for example then hiding it, so wait until you're sure it's a process that will take some considerable amount of time
             */
            delay? : number
            /**
             * Message to display
             */
            message? : string
            /**
             * Render the message as HTML; This can lead to XSS attacks so make sure that you sanitize the message first
             */
            html? : boolean
            /**
             * Spinner size (in pixels)
             */
            spinnerSize? : number
            /**
             * Color name for spinner from the Quasar Color Palette
             */
            spinnerColor? : string
            /**
             * Color name for text from the Quasar Color Palette
             */
            messageColor? : string
            /**
             * Color name for background from the Quasar Color Palette
             */
            backgroundColor? : string
            /**
             * One of the QSpinners
             */
            spinner? : Component
            /**
             * Add a CSS class to easily customize the component
             */
            customClass? : string
            /**
             * Ignore the default configuration (set by setDefaults()) for this instance only
             */
            ignoreDefaults? : boolean }): void

Will the parameter be added in an upcoming release or is deprecated?

Thanks!

**Comment by ontwikkelfabriek** [comment]
@msosa360 

I would create a github issue for this. The 'boxClass' prop should be there:
https://next.quasar.dev/start/upgrade-guide#loading-plugin

**Comment by rstoenescu** [comment, marked as answer]
Fixed it. Will be available in Quasar v2.0.0-rc.2. Thanks for reporting!
